File: WebApp/Scripts/app/model/BG0006M.js
Ext.define('WEBAPP.model.BG0006M', {
    extend: 'Ext.data.Model',
    fields: [
        { name: 'PO_NO', type: 'string' },		//訂單號碼
        { name: 'MMCODE', type: 'string' },		//院內碼
        { name: 'PO_QTY', type: 'string' },		//訂單數量(包裝單位數量)
        { name: 'PO_PRICE', type: 'string' },		//訂單單價(合約單價)
        { name: 'M_PURUN', type: 'string' },		//申購計量單位
        { name: 'M_AGENLAB', type: 'string' },		//廠牌
        { name: 'PO_AMT', type: 'string' },		//總金額
        { name: 'M_DISCPERC', type: 'string' },		//折讓比
        { name: 'DELI_QTY', type: 'string' },		//已交數量
        { name: 'BW_SQTY', type: 'string' },		//借貨數量
        { name: 'DELI_STATUS', type: 'string' },		//交貨狀態 C-已交貨
        { name: 'CREATE_TIME', type: 'string' },		//建立時間
        { name: 'CREATE_USER', type: 'string' },		//建立人員代碼
        { name: 'UPDATE_TIME', type: 'string' },		//異動時間
        { name: 'UPDATE_USER', type: 'string' },		//異動人員代碼
        { name: 'UPDATE_IP', type: 'string' },		//異動IP
        { name: 'MEMO', type: 'string' },		//備註
        { name: 'PR_NO', type: 'string' },		//申購單號
        { name: 'UNIT_SWAP', type: 'string' },		//轉換率
        { name: 'INVOICE', type: 'string' },		//發票號碼
        { name: 'INVOICE_DT', type: 'string' },		//發票號碼日期
        { name: 'CKIN_QTY', type: 'string' },		//發票驗證數量
        { name: 'CHK_USER', type: 'string' },		//發票驗證人員
        { name: 'CHK_DT', type: 'string' },		//發票驗證日期
        { name: 'ACCOUNTDATE', type: 'string' },		//入帳日期
        { name: 'STATUS', type: 'string' },		//default 'N' , D-作廢,    N-申購
        { name: 'UPRICE', type: 'string' },		//最小單價(計量單位單價)
        { name: 'DISC_CPRICE', type: 'string' },		//優惠合約單價
        { name: 'DISC_UPRICE', type: 'string' },		//優惠最小單價(計量單位單價)
        { name: 'CREATE_TIME', type: 'string' },
        { name: 'CREATE_USER', type: 'string' },
        { name: 'UPDATE_TIME', type: 'string' },
        { name: 'UPDATE_USER', type: 'string' },
        { name: 'UPDATE_IP', type: 'string' },
        { name: 'PRICE_AMOUNT', type: 'string' },        //申購金額
        { name: 'ACC_TIME', type: 'string' },        //進貨日期
        { name: 'ACC_QTY', type: 'string' },     //數量
        { name: 'LOT_NO', type: 'string' },     //批號
        { name: 'EXP_DATE', type: 'string' }     //效期
    ]
});

File: WebApp/Models/BG/BG0006M.cs
using System;
using System.Collections.Generic;

namespace WebApp.Models
{
    public class BG0006M : JCLib.Mvc.BaseModel
    {
        public string PO_NO { get; set; }       //訂單號碼
        public string MMCODE { get; set; }      //院內碼
        public string PO_QTY { get; set; }      //訂單數量(包裝單位數量)
        public string PO_PRICE { get; set; }        //訂單單價(合約單價)
        public string M_PURUN { get; set; }     //申購計量單位
        public string M_AGENLAB { get; set; }       //廠牌
        public string PO_AMT { get; set; }      //總金額
        public string M_DISCPERC { get; set; }      //折讓比
        public string DELI_QTY { get; set; }        //已交數量
        public string BW_SQTY { get; set; }     //借貨數量
        public string DELI_STATUS { get; set; }     //交貨狀態 C-已交貨
        public string CREATE_TIME { get; set; }     //建立時間
        public string CREATE_USER { get; set; }     //建立人員代碼
        public string UPDATE_TIME { get; set; }     //異動時間
        public string UPDATE_USER { get; set; }     //異動人員代碼
        public string UPDATE_IP { get; set; }       //異動IP
        public string MEMO { get; set; }        //備註
        public string PR_NO { get; set; }       //申購單號
        public string UNIT_SWAP { get; set; }       //轉換率
        public string INVOICE { get; set; }     //發票號碼
        public string INVOICE_DT { get; set; }      //發票號碼日期
        public string CKIN_QTY { get; set; }        //發票驗證數量
        public string CHK_USER { get; set; }        //發票驗證人員
        public string CHK_DT { get; set; }      //發票驗證日期
        public string ACCOUNTDATE { get; set; }     //入帳日期
        public string STATUS { get; set; }      //default 'N' , D-作廢,    N-申購
        public string UPRICE { get; set; }      //最小單價(計量單位單價)
        public string DISC_CPRICE { get; set; }     //優惠合約單價
        public string DISC_UPRICE { get; set; }		//優惠最小單價(計量單位單價)
        public DateTime PO_DATE { get; set; }
        public string MMNAME_C { get; set; }
        public string MMNAME_E { get; set; }
        public string DISC_AMOUNT { get; set; }
        public string PAY_AMOUNT { get; set; }        
        public string DELI_DT { get; set; }
        public string AGEN_NO { get; set; }
        public string AGEN_NAMEC { get; set; }
        public string M_PHCTNCO { get; set; }
        public string CONTRACNO { get; set; }
        public string M_MATID { get; set; }
        public string PRICE_AMOUNT { get; set; }        //申購金額
        public string ACC_TIME { get; set; }        //進貨日期
        public string ACC_QTY { get; set; }        //數量
        public string LOT_NO { get; set; }        //數量
        public string EXP_DATE { get; set; }        //數量
    }
}